### PR TITLE
fix: some more css tweaks for sidebar

### PR DESCRIPTION
**Pull request by bjoern-michaelsen-freiheit**
Ref: SRX-MVHTKH

### CODE REVIEW

File: services/frontend-service/Earthfile [hunk 1]
```diff
@@ -55,6 +55,7 @@ deps-ui:
     COPY ../../+deps/buf /usr/bin/buf
     COPY ../../pkg+artifacts/pkg/api/v1 api
     COPY src src
+    COPY src src-bak
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/Earthfile:58]: why do we need this?

File: services/frontend-service/Earthfile [hunk 2]
```diff
@@ -67,8 +68,11 @@ node-modules:
 
 lint-ui:
     FROM +deps-ui
-    RUN pnpm eslint-check
-    RUN pnpm scss-check
+    RUN pnpm eslint-fix
+    RUN pnpm scss-fix
+    SAVE ARTIFACT src/ui/components/ServiceLane/ServiceLane.scss AS LOCAL src/ui/components/ServiceLane/ServiceLane.scss
+    SAVE ARTIFACT src/ui/components/SideBar/SideBar.scss AS LOCAL src/ui/components/SideBar/SideBar.scss
+    SAVE ARTIFACT src/ui/components/dropdown/dropdown.scss AS LOCAL src/ui/components/dropdown/dropdown.scss
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/Earthfile:75]: this seems arbitrary. Why only these 3 files?

File: services/frontend-service/Earthfile [hunk 3]
```diff
@@ -67,8 +68,11 @@ node-modules:
 
 lint-ui:
     FROM +deps-ui
-    RUN pnpm eslint-check
-    RUN pnpm scss-check
+    RUN pnpm eslint-fix
+    RUN pnpm scss-fix
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/Earthfile:72]: why are we calling the "fix" targets here? I think in a lint-ui target we should call the lint targets, not the fix targets.

File: services/frontend-service/Earthfile (hunk 1 above)
Comment (bjoern-michaelsen-freiheit) [reply services/frontend-service/Earthfile]: nope, rebase fumble, removed.

File: services/frontend-service/Earthfile (hunk 2 above)
Comment (bjoern-michaelsen-freiheit) [reply services/frontend-service/Earthfile]: rebase fumble, removed

File: services/frontend-service/Earthfile (hunk 3 above)
Comment (bjoern-michaelsen-freiheit) [reply services/frontend-service/Earthfile]: rebase fumble, removed

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

**Pull request merged**